pkgs/development/python-modules/phunspell/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+
+  setuptools,
+
+  spylls,
+}:
+
+buildPythonPackage rec {
+  pname = "phunspell";
+  version = "0.1.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dvwright";
+    repo = "phunspell";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-TlO9Ccr2iRN+s3JV+3P36RF9oFY32fj+24sKslZZCVk=";
+  };
+
+  patches = [
+    (fetchpatch {
+      name = "fix-deprecation-warnings.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/dvwright/phunspell/pull/2.patch";
+      hash = "sha256-uEgMKqtvBn+/xE7Qp2sMpoADm0+NnuCDKA8UEBo/2uA=";
+    })
```

```suggestion
    (fetchpatch {
      name = "fix-package-data-warning.patch";
      url = "https://github.com/dvwright/phunspell/commit/70a0ee8af8442797e03916cea31637c21e6d32d9.patch";
      hash = "sha256-qBVZAHlM4JUXbSYX1bgLFlMmLcK1V6K+lrVthIzGYvU=";
    })
    (fetchpatch {
      name = "replace-deprecated-option.patch";
      url = "https://github.com/dvwright/phunspell/commit/331c593b486cebe1a9b72befa568de9b51033f15.patch";
      hash = "sha256-CUBtDmRV4Da5i+DpeQnOaPXhUN1DBGxUVyRg7bX+Jj8=";
    })
```

using the commits directly here would be safer, as the content of `#2` could change at a later time